<?xml version="1.0" encoding="utf-8" standalone="no"?>
<!--This file represents the results of running a test suite-->
<test-results name="C:\Users\Admin\Desktop\MockProject1\DbMock1G4\TestNUnit\bin\Debug\TestNUnit.dll" total="1" errors="1" failures="0" not-run="0" inconclusive="0" ignored="0" skipped="0" invalid="0" date="2014-09-02" time="23:06:58">
  <environment nunit-version="2.6.3.13283" clr-version="2.0.50727.8000" os-version="Microsoft Windows NT 6.2.9200.0" platform="Win32NT" cwd="C:\Program Files (x86)\NUnit 2.6.3\bin" machine-name="PC" user="Admin" user-domain="PC" />
  <culture-info current-culture="en-US" current-uiculture="en-US" />
  <test-suite type="Assembly" name="C:\Users\Admin\Desktop\MockProject1\DbMock1G4\TestNUnit\bin\Debug\TestNUnit.dll" executed="True" result="Failure" success="False" time="0.302" asserts="0">
    <results>
      <test-suite type="Namespace" name="TestNUnit" executed="True" result="Failure" success="False" time="0.302" asserts="0">
        <results>
          <test-suite type="TestFixture" name="TestWithdraw" executed="True" result="Failure" success="False" time="0.302" asserts="0">
            <results>
              <test-case name="TestNUnit.TestWithdraw.CheckBalance" executed="True" result="Error" success="False" time="0.301" asserts="0">
                <failure>
                  <message><![CDATA[System.TypeInitializationException : The type initializer for 'DbMock1G4.DataAccess.Data' threw an exception.
  ----> System.ArgumentException : Unable to find the requested .Net Framework Data Provider.  It may not be installed.]]></message>
                  <stack-trace><![CDATA[at DbMock1G4.DataAccess.Data.CreateParameter(String parameterName, Object parameterValue)
at DbMock1G4.DataAccess.AccountDA.GetByAccountId(Int32 accountid) in c:\Users\Admin\Desktop\MockProject1\DbMock1G4\DataLayer\AccountDA.cs:line 36
at DbMock1G4.BusinessLogic.AccountBL.GetByAccountId(Int32 accountid) in c:\Users\Admin\Desktop\MockProject1\DbMock1G4\BusinessLogic\AccountBL.cs:line 92
at DbMock1G4.BusinessLogic.AccountBL.CheckBalanceWithDraw(Int32 accId, Decimal money) in c:\Users\Admin\Desktop\MockProject1\DbMock1G4\BusinessLogic\AccountBL.cs:line 28
at TestNUnit.TestWithdraw.CheckBalance() in c:\Users\Admin\Desktop\MockProject1\DbMock1G4\TestNUnit\TestWithdraw.cs:line 32
--ArgumentException
at System.Data.Common.DbProviderFactories.GetFactory(String providerInvariantName)
at DbMock1G4.DataAccess.Data..cctor() in c:\Users\Admin\Desktop\MockProject1\DbMock1G4\DataLayer\Data.cs:line 13
]]></stack-trace>
                </failure>
              </test-case>
            </results>
          </test-suite>
        </results>
      </test-suite>
    </results>
  </test-suite>
</test-results>